room/compiler/src/main/kotlin/androidx/room/processor/CustomConverterProcessor.kt

```diff
@@ -41,14 +42,19 @@ class CustomConverterProcessor(val context: Context, val element: XTypeElement)
         private fun XType.isInvalidReturnType() =
             isError() || isVoid() || isNone()
 
-        fun findConverters(context: Context, element: XElement): ProcessResult {
+        fun findConverters(
+            context: Context,
+            element: XElement,
+            addTypeConverterFactory: (XType) -> Unit = {}
```

Comment:
Lets not have a default value here for `addTypeConverterFactory` it makes sense in the `Context` class but not here since we want to make sure someone gets notified that a type converter factory has to be registered.